s3tests_boto3/functional/test_s3select.py

```diff
@@ -218,6 +223,37 @@ def upload_csv_object(bucket_name,new_key,obj):
         response = c2.get_object(Bucket=bucket_name, Key=new_key)
         eq(response['Body'].read().decode('utf-8'), obj, 's3select error[ downloaded object not equal to uploaded objecy')
 
+def parquet_generator():
+
+    parquet_size = 1000000
+    a=[]
+    for i in range(parquet_size):
+        a.append(int(random.randint(1,10000)))
+
+    b=[]
+    for i in range(parquet_size):
+        b.append(int(random.randint(1,10000)))
+
+    c=[]
+    for i in range(parquet_size):
+        c.append(int(random.randint(1,10000)))
+
+    d=[]
+    for i in range(parquet_size):
+        d.append(int(random.randint(1,10000)))
+
+    df3 = pd.DataFrame({'a': a,
+                   'b': b,
+                   'c': c,
+                   'd': d}
+                   )
+
+
+    table = pa.Table.from_pandas(df3,preserve_index=False)
+
+    print (table)
```

Comment:
@galsalomon66 is there a reason this print statement is in this function?

Reply:
oops, should remove.
its a few lines of the table metadata